patches/chrome-browser-resources-settings-privacy_page-privacy_page.html.patch

```diff
@@ -1,40 +1,56 @@
 diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.html b/chrome/browser/resources/settings/privacy_page/privacy_page.html
-index f58c307b46222efb8acbf2f9bf11af5cf14fd211..639e0e94773bdf68d5b52cb7f8c85e8034b471e4 100644
+index f58c307b46222efb8acbf2f9bf11af5cf14fd211..7a6e204bae3bbbc84ab61e16701f9df4680d4206 100644
 --- a/chrome/browser/resources/settings/privacy_page/privacy_page.html
 +++ b/chrome/browser/resources/settings/privacy_page/privacy_page.html
-@@ -101,6 +101,7 @@
+@@ -38,6 +38,7 @@
+ 
+ <if expr="not chromeos">
+ <link rel="import" href="chrome://resources/cr_elements/cr_toast/cr_toast.html">
++<link rel="import" href="../brave_rewards_page/brave_rewards_page.html">
+ </if>
+ <if expr="use_nss_certs">
+   <link rel="import" href="chrome://resources/cr_components/certificate_manager/certificate_manager.html">
+@@ -101,6 +102,7 @@
             hidden="[[!unifiedConsentEnabled_]]">
          </cr-link-row>
  <if expr="not chromeos">
 +<!--
          <settings-toggle-button id="signinAllowedToggle"
              pref="{{prefs.signin.allowed_on_next_startup}}"
              label="$i18n{signinAllowedTitle}"
-@@ -108,6 +109,7 @@
+@@ -108,6 +110,7 @@
              on-settings-boolean-control-change="onSigninAllowedChange_"
              no-set-pref>
          </settings-toggle-button>
 +-->
  </if><!-- not chromeos -->
          <template is="dom-if" if="[[!unifiedConsentEnabled_]]">
            <settings-personalization-options prefs="{{prefs}}"
-@@ -234,6 +236,7 @@
+@@ -192,6 +195,7 @@
+                 aria-describedby="clearBrowsingDataSecondary"></button>
+           </paper-icon-button-light>
+         </div>
++        <settings-brave-rewards-page prefs="{{prefs}}"></settings-brave-rewards-page>
```

Comment:
I'd like to do this without a patch, via template modification. But since that's not ready let's continue to do it like this. Mentioning it here in case anyone else wants to mention yet another patch line :-)

Reply:
When it is ready, tag me on new issue if needed and I will convert over.